generator/resources/eks_e2e_test_matrix.json

```diff
@@ -0,0 +1,5 @@
+[
+  {
+
+  }
+]
```

Comment:
Each empty set of braces in the JSON array represents the tests under:
```
"eks_e2e": {
		{testDir: "../../../test/e2e/jmx"},
	},
```

So it can't be empty.